docs/sphinx/developers/java-library.txt

```diff
@@ -60,13 +60,48 @@ all be found in our :sourcedir:`Git repository <jar>`.
 Examples of usage
 -----------------
 
+**Reading and Performance:**
+
+:source:`MultiFileExample <components/formats-gpl/utils/MultiFileExample.java>` -
+Simple example of how to open multiple files simultaneously.
+
+:source:`ParallelRead <components/formats-gpl/utils/ParallelRead.java>` -
+Reads all files in given directory in parallel,using a separate thread for each.
+
+:source:`ReadWriteInMemory <components/formats-gpl/utils/ReadWriteInMemory.java>` -
+Tests the Bio-Formats I/O logic to and from byte arrays in memory.
+
+**Writing:**
+
 :source:`MinimumWriter <components/formats-gpl/utils/MinimumWriter.java>` -
 A command line utility demonstrating the minimum amount of metadata needed to
 write a file.
 
+:source:`TiledExport <components/formats-gpl/utils/TiledExportExample.java>` -
+java TiledExportExample
```

Comment:
The ```TiledExportExample``` description could be more informative - maybe something like ```Shows how to convert a file one tile at a time, instead of one plane at a time (needed for very large images).```?